chromium_src/chrome/browser/ui/views/tabs/tab_hover_card_bubble_view.cc

```diff
@@ -25,33 +27,28 @@
 void TabHoverCardBubbleView_ChromiumImpl::BraveUpdateCardContent(
     const Tab* tab) {
   TabHoverCardBubbleView_ChromiumImpl::UpdateCardContent(tab);
-  const std::u16string& domain = domain_label_->GetText();
-  const std::u16string kChromeUISchemeU16 =
-      base::ASCIIToUTF16(base::StrCat({content::kChromeUIScheme, "://"}));
   // Replace chrome:// with brave://. Since this is purely in the UI we can
   // just do a sub-string replacement instead of parsing into GURL.
-  if (base::StartsWith(domain, kChromeUISchemeU16,
-                       base::CompareCase::INSENSITIVE_ASCII)) {
-    std::u16string new_domain = domain;
-    base::ReplaceFirstSubstringAfterOffset(
-        &new_domain, 0ul, kChromeUISchemeU16,
-        base::ASCIIToUTF16(base::StrCat({content::kBraveUIScheme, "://"})));
-    domain_label_->SetData({new_domain, /*is_filename*/ false});
-  }
+  std::u16string domain = domain_label_->GetText();
+  brave_utils::ReplaceChromeToBraveScheme(&domain);
+
+  domain_label_->SetData({domain, /*is_filename*/ false});
```

Comment:
`SetData` should only be called if the replace was performed. Pls make `ReplaceChromeToBraveScheme` return `bool` to detect if a replacement was performed.